### DISCUSSION TITLE
Using InlineCreateOperation with "include_main_form_fields" does not carry over values from select2_multiple fields

### DISCUSSION THREAD

**Discussion by dawolf** [Laravel-Backpack/community-forum#1133] [Q&A (Help)]
Not sure if this is a bug, "by design" or something wrong with my code.
I am trying to use the `InlineCreateOperation` and want to pass the selection from a `select2_multiple` field in the main form over to the InlineCreate form using `include_main_form_fields`.

I want to create an Event that uses the selected Movies from the main form in the InlineCreate form.
Passing the values of `name` (String) and `movies` (n-n Relationship) using `include_main_form_fields` works only for `name`, which I can output in the InlineCreate form. The value for `movies` or any other `select2_multiple` field I try to pass over is just empty.

Any pointers are appreciated.

 ```
 CRUD::field([
            'label' => "Movie",
            'type' => 'select2_multiple',
            'name' => 'movies', // the method that defines the relationship in your Model
            'entity' => 'movies', // the method that defines the relationship in your Model
            'attribute' => 'name', // foreign key attribute that is shown to user
            'pivot' => true, // on create&update, do you need to add/delete pivot table entries?
        ]);

CRUD::field([
            'label' => "Events",
            'type' => 'select2_multiple',
            'name' => 'events', // the method that defines the relationship in your Model
            'entity' => 'events', // the method that defines the relationship in your Model
            'attribute' => 'name', // foreign key attribute that is shown to user
            'pivot' => true, // on create&update, do you need to add/delete pivot table entries?
            'ajax' => true,
            'minimum_input_length' => 0,
            'inline_create' => [ // specify the entity in singular
                'entity' => 'event', // the entity in singular
                'force_select' => true, // should the inline-created entry be immediately selected?
                'add_button_label' => 'New Event', // configure the text for the `+ Add` inline button
                'include_main_form_fields' => ['movies','name'], // pass certain fields from the main form to the modal, get them with: request('main_form_fields')
            ],
        ]);
```

**Comment by karandatwani92** [comment]
Hey @dawolf I have escalated the issue [here](https://github.com/Laravel-Backpack/PRO/issues/276).

Let's continue the discussion there. - Closing this one

**Reply by dawolf** [reply]
Thanks for escalating.
I can't seem to access the PRO repo link. My github email is the same as my backpack account mail. I can not find any information on how to request access to the PRO repo to follow this issue?
